services/movie_session.py

```diff
@@ -0,0 +1,54 @@
+from django.db.models import QuerySet
+from db.models import MovieSession
+from typing import Any
+
+
+def create_movie_session(
+        movie_show_time: Any,
+        movie_id: int = None,
+        cinema_hall_id: int = None
+) -> None:
+    MovieSession.objects.get_or_create(
+        show_time=movie_show_time,
+        cinema_hall_id=cinema_hall_id,
+        movie_id=movie_id,
+    )
+
+
+def get_movies_sessions(session_date: str = None) -> QuerySet:
+    queryset = MovieSession.objects.all()
+
+    if session_date:
+        queryset = queryset.filter(show_time__date=session_date)
+
+    return queryset
+
+
+def get_movie_session_by_id(movie_session_id: int) -> MovieSession:
+    return MovieSession.objects.get(id=movie_session_id)
+
+
+def update_movie_session(
+        session_id: int = None,
+        show_time: str = None,
+        movie_id: str = None,
+        cinema_hall_id: str = None,
+) -> None:

```

Comment:
The parameters `session_id`, `show_time`, `movie_id`, and `cinema_hall_id` have default values of `None`. This could lead to issues if the function is called without arguments. Consider adding validation to ensure these parameters are provided and are valid.